### ISSUE TITLE
figure out how to compute KL divergence effectively

### ISSUE THREAD

**Issue by jettjaniak** [jettjaniak/teren#9]
resid acts are
*batch seq model

logits are
*batch seq vocab

so logits are vocab/model larger than activations
which mean we can't precompute them for all layers and positions

**Comment by jettjaniak** [comment]
we could save SAEExamples to persistent storage and then do the rest of experiments for a single layer at a time
